qiskit_cold_atom/circuit_tools.py

```diff
@@ -0,0 +1,197 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Module to convert cold atom circuits to dictionaries."""
```

Comment:
... dictionaries that can be send to the simulator backends.

Reply:
Sounds good. One question. Why `simulator backends` and not simply `backends`? We would like this to work with hardware too. (unless one calls the hardware a simulator and the "simulator" an emulator but that might be confusing)

Reply:
Done here: 60ecfd4 and here d6de19a

Reply:
Sorry. Of course, `backends` is much better.